README.rst

```diff
@@ -130,7 +136,7 @@ To build the PHP driver for Windows:
 
        .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
-#. Download and install the PHP binaries, or build PHP yourself.
+#. Download and install the PHP binaries, or build the PHP source code.
 
    If you want to build PHP yourself, run the script to download the PHP source and build PHP:
```

Comment:
This one needs to be changed. There is no option to build or not build PHP, it's always needed. So maybe simply say "Run the script to download the PHP source and build PHP"

Reply:
Done